plaid-stl/src/npm/mod.rs

```diff
@@ -0,0 +1,607 @@
+use serde::{Deserialize, Deserializer, Serialize};
+
+use crate::PlaidFunctionError;
+
+use chrono::{DateTime, Utc};
+
+// All the structs which are also used in the Plaid API
+
+/// Permission that is granted to a team over an npm package
+#[derive(Serialize, Deserialize, Clone)]
+pub enum NpmPackagePermission {
+    READ,
+    WRITE,
+}
+
+impl ToString for NpmPackagePermission {
+    fn to_string(&self) -> String {
+        match self {
+            Self::READ => "read".to_string(),
+            Self::WRITE => "write".to_string(),
+        }
+    }
+}
+
+impl NpmPackagePermission {
+    // We do not implement the TryFrom trait because of the error type. We do not want to carry the
```

Comment:
Not sure I follow

Reply:
Please ignore :)
I have now implemented the proper `TryFrom`